addons/account_edi_ubl_cii/data/ubl_20_templates.xml

```diff
@@ -11,7 +11,7 @@
             <cbc:StreetName t-out="vals.get('street_name')"/>
             <cbc:BuildingNumber t-out="vals.get('building_number')"/>
             <cbc:PlotIdentification t-out="vals.get('plot_identification')"/>
-            <cbc:CitySubdivisionName t-out="vals.get('city_subdivision_name ')"/>
+            <cbc:CitySubdivisionName t-out="vals.get('city_subdivision_name')"/>
```

Comment:
Mhh, by rereading it, I don't think we want to do this change in stable. It won't pop a traceback, but a SA company will be forced to update their ubl_cii module to be able to send again this information.